app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -295,7 +295,7 @@ class AttendeeFragment : Fragment(), ComplexBackPressFragment {
     private fun setupCountDownTimer(event: Event) {
         rootView.timeoutCounterLayout.visibility = View.VISIBLE
         rootView.timeoutInfoTextView.text =
-            getString(R.string.ticket_timeout_info_message, event.orderExpiryTime.toString())
+            getString(R.string.ticket_timeout_info_message, COUNT_DOWN_TIME)
```

Comment:
Can you please revert it, solved in PR #2104 otherwise there will be conflicts, also the layout for count down timer